cmd/merlinagent/main.go

```diff
@@ -67,9 +68,12 @@ const (
 
 func main() {
 
+	rand.Seed(time.Now().Unix())
```

Comment:
`rand.Seed(time.Now().UTC().UnixNano())` seems to be a more favored approach. Does `rand.Seed(time.Now().Unix())` have an advantage over the other?

Reply:
Is there any harm moving this to after the command line parsing? My goal is to prevent the application from executing this if a user is only printing the help screen.